packages/widgets/src/Pagination/PageSelect.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import propTypes from '@dhis2/prop-types'
+import i18n from '@dhis2/d2-i18n'
+import { SingleSelect, SingleSelectOption } from '@dhis2/ui-core'
+import { spacers } from '@dhis2/ui-constants'
+import { selectStyles } from './Pagination.styles.js'
+
+const createAvailablePages = length =>
+    Array.from({ length }, (_x, i) => (i + 1).toString())
```

Comment:
I think it's always better to not use abbreviations. They just save a very small amount of coding time but increase the cognitive parsing time every time someone has to read this code, it's prefer the following version:

```js
const createAvailablePages = length =>
    Array
        .from({ length })
        .map((_, index) => (index + 1).toString())
```

To me it's a lot easier to understand what's going on this way

Reply:
I don't feel very strong about this though, if you prefer a short version, just keep it the way it is and resolve this remark

Reply:
I agree with the sentiment. Though on the other hand i is a so common that it wouldn't confuse me as much. I like using an underscore instead of `_x` though. Seems more obvious that it's unused to me.